activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -448,12 +442,18 @@ def load_seed
       #
       # ==== Examples:
       #   ActiveRecord::Tasks::DatabaseTasks.dump_schema_cache(ActiveRecord::Base.connection, "tmp/schema_dump.yaml")
-      def dump_schema_cache(conn, filename)
+      def dump_schema_cache(conn, namespace, schema_cache_path: nil)
```

Comment:
This still has a different API, before it received a filename, now it is a namespace. I think we need to update the example above and also add a CHANGELOG entry since `ActiveRecord::Tasks::DatabaseTasks.dump_schema_cache(ActiveRecord::Base.connection, "tmp/schema_dump.yaml")` will not work anymore.

Reply:
Oh! I completely missed that. Thank you so much for the careful review.

I am going to revert this API back to what it was. I think from @seejohnrun's comments I see a better way of doing it.

Reply:
Hm, ok looking at this more, I will need to change the API of the `ActiveRecord::Tasks::DatabaseTasks.cache_dump_filename` method.

The cache_dump_filename doesn't have any explicit documentation, but it's not :nodoc: or private either, so we definitely will need something in the changelog for that.

Reply:
Wait. No I won't. What am I thinking? The `cache_dump_filename` method already takes the namespace.